nrich-search-spring-boot-starter/README.md

```diff
@@ -66,41 +66,41 @@ nrich.search:
     decimal-number-format-list: #0.00, #0,00
     boolean-true-regex-pattern: ^(?i)\s*(true|yes|da)\s*$
     boolean-false-regex-pattern: ^(?i)\s*(false|no|ne)\s*$
-
 ```
 
 ### Using the module
 
-Module provides `SearchExecutor<T>` interface that enables searching by properties defined in a class and `StringSearchExecutor<T>` interface that enables searching by a string and provided list of
-properties to search. WHen using Hibernate as JPA implementation `NaturalIdSearchExecutor` is also available that enables searching by `@NaturalId` annotated properties.
+`nrich-search` module provides two interfaces:
```

Comment:
Here we say two and below we have three.

Reply:
Ah, an editing mistake.